src/modules/core/WorldModuleBase.js

```diff
@@ -620,18 +620,18 @@ export default class WorldModuleBase extends Eventable {
     const object = component.native;
 
     if (object instanceof Vehicle) {
-      this.execute('removeVehicle', {id: object._physijs.id});
+      this.execute('removeVehicle', {id: component.use('physics').data.id});
```

Comment:
@ianbruce 
1. Maybe let's move `component.use('physics')` to a variable since it is used 5 times in a code?
2. What's the diff between `.data.id` and `.id`?

Reply:
Good idea on abstracting out `component.use('physics')`, and good catch on the id difference, should be `.data.id`; `.id` is undefined